backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt

```diff
@@ -26,7 +27,8 @@ class Entry : CliktCommand() {
     private val config by option().file(canBeDir = false, mustBeReadable = true)
 
     // Options to overwrite single properties of the config
-    private val production by option().choice("true", "false").convert { it.toBoolean() }
+    private val environment by option().choice("dev", "staging", "prod")
```

Comment:
🔧 i would prefer `development` and `production` , we also use this terms in web frontend

Reply:
Okay I will update that. Just thought it might be more handy for terminal commands